docs/topics/native/native-get-started.md

```diff
@@ -1,42 +1,68 @@
-[//]: # (title: Get started with Kotlin/Native in IntelliJ IDEA)
+[//]: # (title: Get started with Kotlin/Native)
 
-This tutorial demonstrates how to use IntelliJ IDEA for creating a Kotlin/Native application.
+In this tutorial, you'll learn how to create a Kotlin/Native application.
 
-To get started, install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
-The tutorial is applicable to both IntelliJ IDEA Community Edition and the Ultimate Edition.
+With Kotlin/Native, you can compile for [different targets](native-target-support.md), including Linux, macOS, and Windows.
+While cross-platform compilation is possible, which means using one platform to compile for a different one,
+in this case, you'll be targeting the same platform you're compiling on.
 
-> If you have a Mac with macOS and want to create and run iOS or desktop applications, you also need to
-> [install Xcode Command Line Tools](https://developer.apple.com/download/). For more information,
-> see [Set up an environment](https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-setup.html).
-> 
+There are several options for getting started with Kotlin/Native:
+
+* The easiest way to visualize the project creation is to [use an IDE](#in-ide).
```

Comment:
```suggestion
* The easiest and most visual way to create a project is by [using an IDE](#in-ide).
```
Visualizing project creation sounds a bit strange to me. I wasn't sure whether you meant being able to explore the project in a visual way or literally that it's a way of creating a project through a UI. I tried to suggest an alternative that sounds more natural. WDYT?